### ISSUE TITLE
Menu is not closed after clicking a link

### ISSUE THREAD

**Issue by emmabckstrm** [osqledaren/web1617#42]

**Comment by tedyw** [comment]
That the menu stays open after route change can be considered as a feature. However some events need to be added to make navigation smooth and intuitive, e.g. auto-closing after some duration of time when mouse pointer has left the menu area and clicking outside the menu closes the menu etc. Some thoughts on this?

**Comment by emmabckstrm** [comment]
Yes, I agree. Auto-closing after some time when mouse pointer has left and clicking outside are good triggers to closing the menu. Maybe scrolling down the page as well?

**Comment by tedyw** [comment]
Auto-closing after some time still needs to be added. It shouldn't close when the mouse pointer is still over the menu area though.

**Comment by tedyw** [comment]
Auto-closing is now working properly. Still considering if menu should close when triggering links on handheld devices. The menu spans across the viewport and it's not easy to see if pages are successfully loaded after links are triggered.
